test/integration/features/basic.feature

```diff
@@ -7,6 +7,7 @@ Feature: Basic
     Given executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
+      And Minishift should have instance config file created
```

Comment:
I think if we want this, we should wait for pull request #540. BTW, you do much more in _shouldHaveInstanceConfig_ than verifying that the config file exist. You also verify its content. This is not apparent here.